nodejs/week1/avg.js

```diff
@@ -0,0 +1,23 @@
+const myArgs = process.argv.slice(2);
+let numArray =[];
+let total = 0;
+let countOfNumbers = 0;
+
+if(myArgs.length === 0) {
+    console.log('Please specify some args the array is empty');
+} else {
+    myArgs.forEach(arg => {
+        const parsedArg = parseFloat(arg);
+        if(!isNaN(parsedArg)) {
+            numArray.push(parsedArg);
+            total += parsedArg;
+            countOfNumbers++;
```

Comment:
Instead of using counter, you could use array length to calculate count. Like myArgs.length should have worked.
Also numArray is bit redundant since you could have used myArgs to print in console log.

Reply:
You can try to do this task with `filter` and `reduce` 😉